scripts/test_examples.sh

```diff
@@ -201,43 +212,255 @@ stopKafka
+")
+
+unexpected=("Error while stopping watcher
+")
+
+cmd="examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf  --offsetMsg 18 --messageCount 0"
+testOutput "${cmd}" "${expected}" "${unexpected}"
+
+# Let us test the running without parameters - in example are generated 10 Kafka Messages to both topics but the consumed is only 5 messages from each topic beginning with offset 5
+expected=("offset arg not set, using default value
+messageCount arg not set, using default value
+Offset: 0, message count: 10
+All plugins initialized successfully
+Sending 10 sync Kafka notifications (protobuf) ...
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '5', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '9', key: 'proto-key', 
+Sending 10 async Kafka notifications (protobuf) ...
+Async message successfully delivered, topic 'example-async-topic', partition '2', offset '0', key: 'async-proto-key'
+Async message successfully delivered, topic 'example-async-topic', partition '2', offset '4', key: 'async-proto-key'
+Async message successfully delivered, topic 'example-async-topic', partition '2', offset '5', key: 'async-proto-key'
+Async message successfully delivered, topic 'example-async-topic', partition '2', offset '9', key: 'async-proto-key'
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '5', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '9', key: 'async-proto-key', 
 Sync watcher closed
 Async watcher closed
 ")
 
+unexpected=("Error while stopping watcher
+")
+
 cmd="examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf"
-testOutput "${cmd}" "${expected}"
+testOutput "${cmd}" "${expected}" "${unexpected}"
+
+# Let us test - in example no new messages generated - we are consummed all beginning with offset 5 till 9  (which were generated before)
+expected=("offset arg not set, using default value
+Offset: 0, message count: 0
+All plugins initialized successfully
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '5', key: 'async-proto-key', 
+Received async Kafka Message, topic 'example-async-topic', partition '2', offset '9', key: 'async-proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '5', key: 'proto-key', 
+Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '9', key: 'proto-key', 
+Sending 0 sync Kafka notifications (protobuf) ...
+Sending 0 async Kafka notifications (protobuf) ...
+Sync watcher closed
+Async watcher closed
+")
+
+unexpected=("Error while stopping watcher
+")
+
+#this is not working - I do not know why
```

Comment:
try  #cmd="examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf -messageCount=0" or remove it